typescript/src/index.ts

```diff
@@ -43,6 +43,8 @@ export class WindowsActivityTracker implements ITracker {
           ts: new Date(),
           windowTitle: res?.title || "[no window selected]",
           process: res?.owner.name || "[no window selected]",
+          processPath: res?.owner.path,
```

Comment:
fine for now. as discussed, let's make this more uniform later: https://github.com/HASEL-UZH/PA.WindowsActivityTracker/issues/10